cli/src/cluster_query.rs

```diff
@@ -768,6 +772,32 @@ pub fn process_catchup(
             )
         };
 
+        let average_time_remaining = if total_sleep_interval == 0 {
+            "".to_string()
+        } else {
+            let distance_slot_variation = start_slot_distance as i64 - slot_distance as i64;
+            let average_catchup_slots_per_second = distance_slot_variation as f64 / f64::from(total_sleep_interval);
```

Comment:
nits: you can remove `catchup` without any loss of information because:

- From context of this code, it's obvious that we're dealing with `catchup`
- There is no other similar variables to disambiguate by adding `catchup`.

Reply:
Below in line 790 I define `average_node_slots_per_second` and in line 794 I use `average_catchup_slots_per_second`

Reply:
ah, Ok, I see the reason now. :)